web/src/features/dashboard/components/hooks.ts

```diff
@@ -46,17 +46,35 @@ type ChartData = {
   value?: number;
 };
 
+type FieldMappingItem = {
+  uniqueIdentifierColumns: {
+    accessor: string;
+    formatFct?: (value: string) => string;
+  }[];
+  valueColumn: string;
+};
+
+function getLabelValue(
```

Comment:
can you add a breif comment on what this does?

Reply:
Change name of function